src/main/java/com/onwelo/tinvalidator/validator/BEValidator.java

```diff
@@ -4,21 +4,20 @@
 
 public class BEValidator implements Validator {
 
-    private static final Pattern PATTERN = Pattern.compile("[0-9]{11}");
+    private static final Pattern PATTERN = Pattern.compile("0?[1-9][0-9]{8}");
```

Comment:
Według dokuemntu nie powinno być raczej `0[1-9][0-9]{8}`. Pytanie czy wynika to z przeoczenia czy jest gdzieś jeszcze inny dokument z innym algorytmem? Jeśli jest warto byłoby go pewnie podlinkować.

Reply:
wiodące zero jest opcjonalne